gateway-ha/src/test/java/io/trino/gateway/CustomImageNameSubstitutor.java

```diff
@@ -0,0 +1,38 @@
+/*
```

Comment:
> allow customization of test containers image name

Please capitalize the commit title. https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages